### ISSUE TITLE
API returns "modpacks":[]

### ISSUE THREAD

**Issue by ghost** [TechnicPack/TechnicSolder#36]
TL;DR Details
- No Solder mod packs appear on Platform 'create mod pack' page despite successfully linking
- Clients do not download pack image resources from repo

These features used to work--I cannot identify what change broke them. All other features appear fine: Platform will link after I have created a pack and it sees my Solder has identically named pack. I have rebuilt my set up from scratch to no avail. I have infinite patience, and time is not an issue.
- Apache 2.4.4 || PHP 5.4.16 || PDO SQLite for database
- Using most recent (6/14) master branch zip download of Solder

I read on another issue that SQLite might be broken--I will try implementing MySQL. Any addition information can be provided gladly. I'll try asking the IRC channel.
